src/containers/User/UserArea.tsx

```diff
@@ -1,47 +1,73 @@
-import React from 'react'
-import { Button, Grid, Label, Segment, Sticky } from 'semantic-ui-react'
+import React, { useEffect, useState } from 'react'
+import { Button, Grid, Header, Label, Menu, MenuProps, Segment, Sticky } from 'semantic-ui-react'
 import strings from '../../utils/constants'
 import { useUserState } from '../../contexts/UserState'
 import UserSelection from './UserSelection'
 import { useRouter } from '../../utils/hooks/useRouter'
+import useListTemplates from '../../utils/hooks/useListTemplates'
 
 const UserArea: React.FC = () => {
   const {
-    userState: { currentUser },
+    userState: { currentUser, isLoading, templatePermissions },
     logout,
   } = useUserState()
-  const { replace } = useRouter()
+  const {
+    replace,
+    push,
+    query: { type },
+  } = useRouter()
+  const [menuItems, setMenuItems] = useState<MenuProps>()
+  const { error, loading, filteredTemplates } = useListTemplates(templatePermissions, isLoading)
+
+  useEffect(() => {
+    if (filteredTemplates) {
+      const items = filteredTemplates.map(({ name, code, permissions }) => ({
+        key: `app_menu_${name}`,
+        content: <Header as="h5" inverted content={name} />,
+        active: type === code,
+        onClick: () => push(`/applications?type=${code}&user-role=${permissions[0]}`),
+      }))
+      const homeItem = {
+        key: `app_menu_home`,
+        content: <Header as="h5" inverted icon="home" />,
+        active: location.pathname === '/',
+        onClick: () => push('/'),
+      }
+      setMenuItems({ items: [homeItem, ...items] })
+    }
+  }, [loading, filteredTemplates, type])
 
   const handleLogOut = async () => {
     await logout()
     replace('/login')
   }
 
-  return currentUser ? (
+  return (
     <Sticky>
       <Segment inverted vertical>
         <Grid inverted>
-          <Grid.Column style={{ width: '30%' }}>
+          <Grid.Column style={{ width: '80%' }}>
+            {error}
+            {menuItems && <Menu fluid inverted pointing secondary {...menuItems} />}
             <Segment inverted>{strings.TITLE_COMPANY_PLACEHOLDER}</Segment>
           </Grid.Column>
-          <Grid.Column style={{ width: '50%' }}>
-            <Segment inverted />
-          </Grid.Column>
           <Grid.Column style={{ width: '20%' }}>
```

Comment:
Same here. Isn't the total "width" in semantic units 16? So the first column should be `width={12}` and this one `width={4}`?

Reply:
Done: [441d675](https://github.com/openmsupply/application-manager-web-app/pull/262/commits/441d67506d3f338022fe9fe4914467d424cc610b)